src/style/_layout.scss

```diff
@@ -281,7 +281,7 @@
         font-weight: bold;
 
         min-width: 80px;
-        padding: 6px 15px;
+        padding: 6px;
```

Comment:
What would you say to this suggestion? As the method name is already centered we technically don't need any left/right padding. Following suggestion set the left/right padding to `0` instead of `6px`.

```suggestion
        padding: 6px 0;
```